CanalTP/SamEcoreUserManagerBundle
# Adding timezone for users

Default timezone is "Europe/Paris"

To test : 
- Update your sam-ecore-user-manager-bundle
- execute from console : 
  - ```./app/console claroline:migration:upgrade CanalTPSamEcoreUserManagerBundle ```
- You should see timezone field on user add / edit page.
- Europe/Paris should be selected
